index.html

```diff
@@ -709,8 +709,8 @@ <h3 id="ot_selectHardwareAcceleration">
 												aria-describedby="ot_decriptionHardwareAcceleration">
 												<div class="row ot_hardwareAcceleration">
 													<div class="col r-option version" role="option" tabindex="0"
-														aria-selected="true" id="ot_CUDA">
-														<span><abbr>CUDA</abbr></span>
+														aria-selected="true" id="ot_CUDA118">
+														<span><abbr>CUDA 11.8</abbr></span>
```

Comment:
> updates CUDA option in training to be a specific CUDA version

Is this necessary? This will require us to update the page every time we release ort with support for a new cuda version.

Reply:
Let's keep this, it makes it clear the version we support. Also, in the next release we will publish packages for 11.8 and 12.1 so users will need a way to choose the cuda version.